core/src/main/resources/defaults/clustertemplates/7.2.15/aws/rt-datamart.json

```diff
@@ -99,8 +99,8 @@
           "attachedVolumes": [
             {
               "count": 2,
-              "size": 1900,
-              "type": "ephemeral"
+              "size": 1000,
```

Comment:
2 TB storage per executor is likely to small for Kudu.  Azure and GCP provision 7 TB based on previous guidance from the Kudu team.  See my comments in the JIRA for more background.  The Kudu team (Kurt Deschler) should provide a definitive recommendation on this.

Reply:
The size of the storage depends on the customer's expected data storage. If they have flexibility to add more disks or expand disks later on we should be good with 3 disks of each 1 TB size as we stripe data across 3 disks in Kudu by default for more parallelism.
Also can you point me to the discussion where 7TB recommendation by the team?

Reply:
The Kudu team originally configured AWS with 3.8 TB of ephemeral and configured Azure and GCP with 7 TB.  I had confirmed the 7 TB size with Grant Henke (former Kudu lead) when the GCP template was released (see my comment in the JIRA).  The current Kudu team (Kurt Deschler and Manish Maheshwari) should make the final decision on this.